README.md

```diff
@@ -1882,6 +1882,7 @@ Update Time, five active automations, webhooks.
   * [Tiledesk](https://tiledesk.com) - Create chatbots and conversational apps. Bring them omnichannel: from your website (live chat widget) to WhatsApp. Free plan with unlimited chatbots.
   * [Versionfeeds](https://versionfeeds.com) — Custom RSS feeds for releases of your favorite software. Have the latest versions of your programming languages, libraries, or loved tools in one feed. (The first 3 feeds are free)
   * [videoinu](https://videoinu.com) — Create and edit screen recordings and other videos online.
+  * [Screenshots to Help Docs](https://myaskai.com/customer-support-help-docs-with-saas-screenshots) — Create free customer support Help Docs by simply uploading your SaaS product screenshots
```

Comment:
Dont know, this looks like a minor promotional thing built with the larger offering, mainly as bait to upgrade. Seems like something that will just be removed soon again, I'll pass.

And will probably soon remove most new AI services from the list also for same reasons.